<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdvancedSearch</class>
 <widget class="QWidget" name="AdvancedSearch">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>537</width>
    <height>419</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Advanced search</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>&amp;Types</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QCheckBox" name="useType1">
          <property name="text">
           <string>Type 1</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="type1"/>
        </item>
        <item row="1" column="0">
         <widget class="QCheckBox" name="useType2">
          <property name="text">
           <string>Type 2</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="type2"/>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>&amp;Ability</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QComboBox" name="ability"/>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="title">
        <string>&amp;Base Stats</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="hplabel">
          <property name="text">
           <string>Life Points</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="hpbox">
          <item>
           <property name="text">
            <string/>
           </property>
          </item>
          <item>
           <property name="text">
            <string>&gt;=</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>=</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>&lt;=</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLineEdit" name="hpedit">
          <property name="minimumSize">
           <size>
            <width>35</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="atklabel">
          <property name="text">
           <string>Attack</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="atkbox">
          <item>
           <property name="text">
            <string/>
           </property>
          </item>
          <item>
           <property name="text">
            <string>&gt;=</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>=</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>&lt;=</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QLineEdit" name="atkedit">
          <property name="minimumSize">
           <size>
            <width>35</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="deflabel">
          <property name="text">
           <string>Defense</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QComboBox" name="defbox">
          <item>
           <property name="text">
            <string/>
           </property>
          </item>
          <item>
           <property name="text">
            <string>&gt;=</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>=</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>&lt;=</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QLineEdit" name="defedit">
          <property name="minimumSize">
           <size>
            <width>35</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="spatklabel">
          <property name="text">
           <string>Sp. Attack</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QComboBox" name="spatkbox">
          <item>
           <property name="text">
            <string/>
           </property>
          </item>
          <item>
           <property name="text">
            <string>&gt;=</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>=</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>&lt;=</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QLineEdit" name="spatkedit">
          <property name="minimumSize">
           <size>
            <width>35</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="spdeflabel">
          <property name="text">
           <string>Sp. Defense</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QComboBox" name="spdefbox">
          <item>
           <property name="text">
            <string/>
           </property>
          </item>
          <item>
           <property name="text">
            <string>&gt;=</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>=</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>&lt;=</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="4" column="2">
         <widget class="QLineEdit" name="spdefedit">
          <property name="minimumSize">
           <size>
            <width>35</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="speedlabel">
          <property name="text">
           <string>Speed</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QComboBox" name="speedbox">
          <item>
           <property name="text">
            <string/>
           </property>
          </item>
          <item>
           <property name="text">
            <string>&gt;=</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>=</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>&lt;=</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="5" column="2">
         <widget class="QLineEdit" name="speededit">
          <property name="minimumSize">
           <size>
            <width>35</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QGroupBox" name="groupBox_4">
       <property name="title">
        <string>&amp;Moves</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QLineEdit" name="move1"/>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="move2"/>
        </item>
        <item row="1" column="0">
         <widget class="QLineEdit" name="move3"/>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="move4"/>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="search">
       <property name="text">
        <string>&amp;Search !</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_5">
       <property name="title">
        <string>&amp;Results</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QListWidget" name="results"/>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
